.github/workflows/release.yml

```diff
@@ -0,0 +1,55 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Go
+on:
+  push:
+    tags:
+      - "v1.0.0-*"
```

Comment:
Is this regex correct?

It will match `v1.0.0` but not `v1.0.1`. What is the intent here?

Reply:
I think you want to match anything > v1 using semver

Reply:
Updated the pattern to match all versions > v1.